supabase.sql

```diff
@@ -0,0 +1,61 @@
+-- Create a table for Public Profiles
+create table profiles (
+  id uuid references auth.users not null,
+  updated_at timestamp with time zone,
+  username text unique,
+  avatar_url text,
+  website text,
+
+  primary key (id),
+  unique(username),
+  constraint username_length check (char_length(username) >= 3)
+);
+
+alter table profiles
+  enable row level security;
+
+create policy "Public profiles are viewable by everyone." on profiles
+  for select using (true);
+
+create policy "Users can insert their own profile." on profiles
+  for insert with check (auth.uid() = id);
+
+create policy "Users can update own profile." on profiles
+  for update using (auth.uid() = id);
+
+-- Set up Realtime!
+begin;
+  drop publication if exists supabase_realtime;
+  create publication supabase_realtime;
```

Comment:
why do we need a realtime db here?

Reply:
This is from the starter template.

Reply:
if we don't need realtime yet, let's remove it

Reply:
@iShibi I'm coming back to this part of the code to re-check it. Where did you get the starter template - was it from the SB dashboard?

Reply:
@ykdojo yes. `SQL Editor` -> `Welcome` -> `User Management Starter`